resourcemgr/resourcemgr.c

```diff
@@ -1309,7 +1309,8 @@ TSS2_RC ResourceMgrSendTpmCommand(
                 goto SendCommand;
             }
 
-            if( foundEntryPtr->connectionId != cmdConnectionId )
+            if( foundEntryPtr->status.persistent == 0 &&
+                foundEntryPtr->connectionId != cmdConnectionId )
```

Comment:
agree, but this needs to be changed in the RM spec first.  If you would like to attend one of the TSS WG meetings and present your case for this, we could see what the WG says.